Assets/Scripts/MovePB.cs

```diff
@@ -37,8 +39,12 @@ private void FixedUpdate()
         _userRot += new Vector3(0, _rotationInput, 0);
 
         _transform.rotation = Quaternion.Euler(_userRot);
-        _rigidbody.velocity += transform.forward * _playerInput * InputScale;
-        
+
+        // Up is always z so velocity of x and z is clamped down
+        if (euclideanNorm(_rigidbody.velocity.x,
+                          _rigidbody.velocity.z) < MaxSpeed)
```

Comment:
As a style thing, I would use the parentheses {} for any if statement, even 1-liner. This is because, if someone adds code later, they may forget to add the {} and this would result in a bug.

Reply:
Sure